### PR TITLE
EZP-31266: Changing items order in Relation List doesn't work on Firefox

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-31266](https://jira.ez.no/browse/EZP-31266)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Firefox for a long time has this error. The arrows aren't elements in their own right, so they can't be focused, therefore need to force focus on input when value changes

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js [hunk 1]
```diff
@@ -215,7 +215,15 @@
...
         };
         const attachRowsEventHandlers = () => {
-            findOrderInputs().forEach((item) => item.addEventListener('blur', updateSelectedItemsOrder, false));
+            findOrderInputs().forEach((item) => {
+                item.addEventListener('blur', updateSelectedItemsOrder, false);
+                item.addEventListener('change', focusOnElement, false);
+            });
+        };
+        const focusOnElement = (event) => {
+            if(doc.activeElement !== event.target) {
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js]: please install Prettier and use it on the file

Comment (mateuszdebinski) [reply src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js]: Done

File: src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js [hunk 2]
```diff
@@ -215,7 +215,15 @@
             return [...relationsContainer.querySelectorAll('[type="checkbox"]')];
         };
         const attachRowsEventHandlers = () => {
-            findOrderInputs().forEach((item) => item.addEventListener('blur', updateSelectedItemsOrder, false));
+            findOrderInputs().forEach((item) => {
+                item.addEventListener('blur', updateSelectedItemsOrder, false);
+                item.addEventListener('change', focusOnElement, false);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js]: If this is only on FF I don't see a point to attach this event in all browsers. Please add a check if it is FF and attach it only in this case.

Comment (mateuszdebinski) [reply src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js]: the added condition only for FF attach

File: src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js [hunk 3]
```diff
@@ -215,7 +215,18 @@
             return [...relationsContainer.querySelectorAll('[type="checkbox"]')];
         };
         const attachRowsEventHandlers = () => {
-            findOrderInputs().forEach((item) => item.addEventListener('blur', updateSelectedItemsOrder, false));
+            const isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js]:
```suggestion
            const isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
            
```

File: src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js [hunk 4]
```diff
@@ -215,7 +215,18 @@
...
+            findOrderInputs().forEach((item) => {
+                item.addEventListener('blur', updateSelectedItemsOrder, false);
+                if (isFirefox) {
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js]:
```suggestion

                if (isFirefox) {
```

**Comment by mateuszbieniek** [comment]
Why add prettier to packages.json?

**Comment by mateuszdebinski** [comment]
> Why add prettier to packages.json?

Mistake, when I install prettier and I didn't notice that package.json was added as well

**Comment by konradoboza** [comment]
@adamwojs please merge. :)

**Pull request merged**